tools/generators/epic-generator/index.ts

```diff
@@ -0,0 +1,186 @@
+import { getProjects, stripIndents, Tree } from '@nrwl/devkit';
+import { access, readFileSync } from 'fs';
+import { execSync } from 'child_process';
+import fetch from 'node-fetch';
+import { EpicGeneratorSchema } from './schema';
+
+function validateSchema(schema: EpicGeneratorSchema) {
+  if (!schema.repository.match(/[A-z-]+\/[A-z-]+/)) {
+    throw new Error('Must provide a valid repository for the issue');
+  }
+
+  if ((schema.title || '').trim() === '') {
+    throw new Error('Must provide a valid title for the issue');
+  }
+
+  return {
+    repository: schema.repository,
+    title: schema.title,
+    message: schema.message || '*Description to be added*',
+  };
+}
+
+const checkAuthentication = async () => {
+  const authStatus = execSync(`gh auth status`).toString();
+
+  if (authStatus.search('You are not logged into any GitHub hosts') > 0) {
+    throw new Error('You are not logged into GitHub CLI (gh).');
+  }
+};
+
+interface Package {
+  name: string;
+  folder: string;
+  owners?: string[];
+}
+
+interface Ownership {
+  path: string;
+  owners: string[];
+}
+
+interface MigrationIssue {
+  issueUrl?: string;
+  assignee?: string;
+  packages: Package[];
+}
+
+interface MigrationIssueMap {
+  [key: string]: MigrationIssue;
+}
+
+const getConvergencePackages = (tree: Tree) => {
```

Comment:
```suggestion
const getConvergedPackages = (tree: Tree) => {
```

nit converged package tends to be the term we generally use